Update README and CHANGELOG

goodmami opened the pull request:
This expands the README.md with more information and links, and renames and reformats update_log to CHANGELOG.md. It also adds PyPI project classifiers and metadata so the README is properly rendered at https://pypi.org/project/smatch/.

Notes:
* This PR touches `setup.py`, so it will probably conflict once #23 is merged; I can rebase at that time
* The Python-version badge won't show properly until a release is uploaded to PyPI including the Python version classifiers (included in this commit in `setup.py`)
* When preparing the release artifacts and uploading to PyPI, make sure to have the following software versions in your environment (see [here](https://packaging.python.org/guides/making-a-pypi-friendly-readme/)):
  - `setuptools >= 38.6.0`
  - `wheel >= 0.31.0`
  - `twine >= 1.11.0`

goodmami committed:
Update README and CHANGELOG

This expands the README.md with more information and links, and
renames and reformats update_log to CHANGELOG.md. It also adds PyPI
project classifiers and metadata so the README is properly rendered at
https://pypi.org/project/smatch/.

snowblink14 commented on README.md:
```diff
@@ -1,7 +1,108 @@
-The code was mostly developed during 2012-2013, and has undergone many fixes and updates. It is now hosted on github for better collaboration.
+[Smatch](http://amr.isi.edu/evaluation.html) is an evaluation tool for
+[AMR](http://amr.isi.edu/) (Abstract Meaning Representation). It
+estimates the similarity of two AMR graphs in terms of their matching
```
estimates->computes
similarity->smatch score (defined below)

snowblink14 commented on README.md:
```diff
@@ -1,7 +1,108 @@
+* [freesunshine0316/sembleu](https://github.com/freesunshine0316/sembleu)
+  is inspired by BLEU and puts more weight on "content" than
+  graph-structure similarity
+* https://github.com/rafaelanchieta/sema/ weights error types
```
[rafaelanchieta/sema](https..)?

snowblink14 commented on setup.py:
```diff
@@ -17,10 +17,28 @@
+          'Topic :: Text Processing :: Linguistic',
+          'Topic :: Utilities'
+      ],
+      keywords='nlp semantics amr',
```
add "evaluation" as a keyword?

snowblink14 reviewed (commented):
Some trivial comments.

goodmami committed:
Resolve comments from PR review

- Adjust wording in README.md intro
- Fix link text in README.md's related projects
- Add "evaluation" as a keyword in `setup.py`

goodmami commented:
Thanks for the review. I've resolved your comments.

snowblink14 reviewed (approved):
Thanks for making the change! Overall LGTM. You might want to sync with @danielhers on the submission order regarding #23.

goodmami commented:
Thanks!

> You might want to sync with @danielhers on the submission order regarding #23.

As soon as #23 is merged I will rebase. The changes are trivial, I think.

goodmami committed:
Swap order of test/gold in README

goodmami commented:
I just noticed I had the order of "test" and "gold" backwards in the new README, which is now fixed in the last commit.

@danielhers: @snowblink14 said I should coordinate with you on merging #23. I'm happy to rebase my branch after yours is merged. If it's ready to go, can you merge or does @snowblink14 need to?

danielhers commented:
I don't have push access to this repository. @snowblink14, could you merge
#23 please?

tir. 7. jan. 2020 03.54 skrev Michael Wayne Goodman <
notifications@github.com>:

> I just noticed I had the order of "test" and "gold" backwards in the new
> README, which is now fixed in the last commit.
>
> @danielhers <https://github.com/danielhers>: @snowblink14
> <https://github.com/snowblink14> said I should coordinate with you on
> merging #23 <https://github.com/snowblink14/smatch/pull/23>. I'm happy to
> rebase my branch after yours is merged. If it's ready to go, can you merge
> or does @snowblink14 <https://github.com/snowblink14> need to?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/snowblink14/smatch/pull/24?email_source=notifications&email_token=AACAJQ6B2VO6ZENZJNGK6NDQ4PVG7A5CNFSM4J7IOGY2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIHQQUY#issuecomment-571410515>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACAJQZCLL2TYK4N6YQDD33Q4PVG7ANCNFSM4J7IOGYQ>
> .
>

snowblink14 commented:
> I don't have push access to this repository. @snowblink14, could you merge #23 please? tir. 7. jan. 2020 03.54 skrev Michael Wayne Goodman < notifications@github.com>:
> […](#)
> I just noticed I had the order of "test" and "gold" backwards in the new README, which is now fixed in the last commit. @danielhers <https://github.com/danielhers>: @snowblink14 <https://github.com/snowblink14> said I should coordinate with you on merging #23 <#23>. I'm happy to rebase my branch after yours is merged. If it's ready to go, can you merge or does @snowblink14 <https://github.com/snowblink14> need to? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <#24?email_source=notifications&email_token=AACAJQ6B2VO6ZENZJNGK6NDQ4PVG7A5CNFSM4J7IOGY2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIHQQUY#issuecomment-571410515>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AACAJQZCLL2TYK4N6YQDD33Q4PVG7ANCNFSM4J7IOGYQ> .

Just merged #23.

goodmami committed:
Remove unused imports from setup.py

goodmami commented:
@snowblink14 ok, this is now rebased to the current master branch. There were no conflicts in the end, but since @danielhers partially removed Python 2 support in his PR, I went ahead and removed some now unused imports in setup.py in the last commit.

goodmami committed:
Update CHANGELOG.md with recent commits

snowblink14 commented:
Shall I merge this? @goodmami

goodmami committed:
Update CHANGELOG; fix PyPI link

goodmami commented:
@snowblink14 sure, i've kept it fresh so anytime is good. In any case, I just fixed a link in the CHANGELOG.md and added a new entry for the latest merge, but these are just documentation changes.
